pages/windows/ospp.vbs.md

```diff
@@ -0,0 +1,29 @@
+# ospp.vbs
+
+> Install, activate, and manage volume licensed versions of Office products.
+> Note: this command may override, deactivate, and/or remove your current volume of licensed Office product versions, so please proceed cautiously.
+> More information: <https://learn.microsoft.com/deployoffice/vlactivation/tools-to-manage-volume-activation-of-office>.
+
+- Install a product key (Note: it replaces the existing key):
+
+`cscript ospp.vbs /inpkey:{{product_key}}`
+
+- Uninstall an installed product key with the last five digits of the product key:
+
+`cscript ospp.vbs /unpkey:{{product_key}}`
```

Comment:
```suggestion
`cscript ospp.vbs /unpkey:{{product_key_digits}}`
```
Just in case anyone takes a look at the command first and then read the associated description.